tools/depends/configure.ac

```diff
@@ -248,6 +248,9 @@ case $host in
       arm*-*linux-android*)
         if test "x$use_cpu" = "xauto"; then
           use_cpu="armeabi-v7a"
+          use_cmake_cpu=armv7-a
+        else
+          use_cmake_cpu=armv6
```

Comment:
Is armv6 still supported?

Reply:
Was thought as minimum callback, heard armv6 is almost identical to armv7.

Reply:
But does Kodi (and all it's dependencies) still support armv6?

Reply:
I can leave away, no problem, what you thing is better.

To force `use_cmake_cpu=armv7-a` for all (where I think then the best) or make `use_cmake_cpu=$host_cpu`?

Is confusing for me that there still `use_cpu` is checked for `auto` then.

Reply:
Becomes forced to `armv7-a`, the Android docs say that on API only `armeabi-v7a` is supported and others removed on NDK r17.